packages/server/lib/controllers/account.controller.ts

```diff
@@ -128,15 +137,13 @@ class AccountController {
                 connection_id: 'n/a',
                 provider: null,
                 provider_config_key: '',
-                //environment_id: result.environmentId
                 environment_id: response.environment.id
             };
 
             await createActivityLogAndLogMessage(log, {
                 level: 'info',
                 environment_id: response.environment.id,
                 timestamp: Date.now(),
-                //content: `A Nango admin logged into your account for the following reason: "${login_reason}"`
                 content: `A Nango admin logged into another account for the following reason: "${login_reason}"`
```

Comment:
I still think we don't want to show this messaging which is why it was commented out. So you can delete this completely.

Reply:
For compliance it's often required to log something, maybe @bastienbeurier have an opinion on this?

Reply:
I wanted to have this messaging here as well, which is why I commented it out and didn't delete it FWIW

Reply:
I agree we should log this at some point. I don't mind whether it's now or later. I slightly prefer waiting for our project on logs DX where we'll redefine everything that needs to be logged.

Reply:
It still should be “your” account if we do show it

Reply:
alright let's not touch this and rediscuss ☺️